_posts/2025-07-22-openvox-releases.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: New OpenVoxProject 8 releases!
+date: 2025-07-31
+github_username: bastelfreak
+---
+
+## New OpenVoxProject 8 releases!
+
+After lots of development, we have a new round of releases available!
+
+Our last releases are:
+
+* openvox-agent [8.19.2](https://github.com/OpenVoxProject/openvox-agent/releases/tag/8.19.2)
+* openvox-server [8.8.1](https://github.com/OpenVoxProject/openvox-server/blob/main/CHANGELOG.md#881-2025-03-19)
+* openvoxdb [8.9.1](https://github.com/OpenVoxProject/openvoxdb/blob/main/CHANGELOG.md#891)
+
+Our new releases:
+
+* openvox-agent [9.21.1](https://github.com/OpenVoxProject/openvox-agent/releases/tag/8.21.1)
+* openvox-server [8.10.0](https://github.com/OpenVoxProject/openvox-server/releases/tag/8.10.0)
+* openvoxdb will soon follow
+
+
+## Where are the releases?
+
+Right now, those are *beta* releases.
```

Comment:
Maybe a note here saying why they are beta, because this is the first set using openfact, which required some packaging tooling changes, and we want to make sure it's fully solid before releasing to the apt/yum repos.

Reply:
I rephrased it